src/amo/components/SecondaryHero/styles.scss

```diff
@@ -35,6 +35,7 @@
   font-weight: normal;
   line-height: 1.333;
   margin: 0;
+  word-break: break-word;
```

Comment:
How about using something else given your comment in https://github.com/mozilla/addons-frontend/pull/8523#issuecomment-535646716? (and also https://github.com/mozilla/addons-frontend/issues/8396)

Reply:
I really should follow my own advice, shouldn't I? 😛 

I tried `overflow-wrap: break-word`, but it doesn't work on this page either, so I changed it to `overflow-wrap: anywhere` as discussed at https://github.com/mozilla/addons-frontend/pull/8523#issuecomment-535646716.